AirCasting/CoreData/ExternalSessionEntity+CoreDataProperties.swift

```diff
@@ -49,3 +66,89 @@ extension ExternalSessionEntity {
 extension ExternalSessionEntity : Identifiable {
 
 }
+
+extension ExternalSessionEntity {
+
+    var pm1Stream: MeasurementStreamEntity? {
+        let match = measurementStreams?.first(where: { (stream) -> Bool in
+            (stream as? MeasurementStreamEntity)?.sensorName == "AirBeam3-PM1" ||
+                (stream as? MeasurementStreamEntity)?.sensorName == "AirBeam2-PM1" ||
+                    (stream as? MeasurementStreamEntity)?.sensorName == "AirBeam1-PM1"
+        })
```

Comment:
Why we are not using the same approach as below to find those that `.contains("PM1)`?

Reply:
Because PM10 also contains PM1